### ISSUE TITLE
Turkish character error

### ISSUE THREAD

**Issue by emresaracoglu** [projectsend/projectsend#650]
Hello,

Thank your for your great work! 
Customer names and descriptions have a Turkish character error.

`Emre Sara&ccedil;oğlu`  _(Emre Saraçoğlu)_

How can I fix this?

### Environment

* ProjectSend version: r1053
* php version: 7.1.10
* MySQL version: 10.2.18-MariaDB

**Comment by manoocher** [comment]
Change the database Collation to utf8_unicode_ci, the correct encoding applies to only new entries. This post is old but it can be used as reference new searches.

**Comment by emresaracoglu** [comment]
Yes, but I think database collation should be created by default `utf8_unicode_ci`

**Comment by ignacionelson** [comment]
I've had great results with utf8mb4_general_ci, but had a hard time making a script that would update the collations AND content for existing users :(

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!

**Comment by emresaracoglu** [comment]
@ignacionelson Thank you for your work.
